src/js/angular/rest/graph-data.rest.service.js

```diff
@@ -136,4 +153,32 @@ function GraphDataRestService($http) {
             }
         });
     }
+
+    function resolveGraphs(scope, selectedGraph) {
+        if (scope.getActiveRepository() !== "") {
```

Comment:
As long as this service method is called in a promise, this check here might cause the promise to be resolved with `undefined`.
Which might be OK actually as you don't actually expect a return value at all due to the side effect on the scope.